packs/feats/overdrive-ally.json

```
@@ -28,7 +28,28 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.overdriveAlly.damage",
```

```suggestion
                "path": "flags.pf2e.inventor.overdriveAlly.damage",
```
Best to keep things organized.